sites/main-site/src/content/blog/2025/paper_v2.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Empowering bioinformatics communities with Nextflow and nf-core"
+subtitle: "Genome Biology review"
+pubDate: 2025-08-05T12:00:00+01:00
+headerImage: "/assets/images/blog/paper_v2/paper_v2.png"
+headerImageAlt: "nf-core logo"
+headerImageDim: [250, 250]
+embedHeaderImage: false
+authors:
+    - "maxulysse"
+    - "jfy133"
+    - "mashehu"
+label:
+    - "maintainers"
+---
+
+## Empowering bioinformatics communities with Nextflow and nf-core
+
+Last year, we [announced](https://bsky.app/profile/nf-co.re/post/3ksi4mr6uyl25) the preprint, but we are now excited to announce that the latest nf-core community paper in **Genome Biology** is live!
+
+[Empowering bioinformatics communities with Nextflow and nf-core](https://rdcu.be/eyqzX)
+
+This is a collaborative paper with EuroFAANG, where we have summarised the latest updates and progress within Nextflow and nf-core (e.g. the now standardised DSL2 syntax and the modules repository).
+The collaboration with EuroFAANG has been a fruitful one that extended all things **FAIR**, and lead to the creation of the [Special Interest Groups](https://nf-co.re/blog/2024/special_interest_groups) at nf-core.
+
+We would like to thank Björn E. Langer for leading the whole thing and all the community members without whom this would not have been possible.
```

Comment:
```suggestion
We would like to thank Björn E. Langer for leading this effort and all the community members without whom this would not have been possible.
```